buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 16
+  build:
+    commands:
+      - echo This is ${ENV}
+      - echo ${URL} # defined in BuildProject"
+      - NEXT_PUBLIC_DESIGN_URL=${NEXT_PUBLIC_DESIGN_URL}
+      - NEXT_PUBLIC_HEALTH_API=${NEXT_PUBLIC_HEALTH_API}
+      - NEXT_PUBLIC_API_BASE_URI=${NEXT_PUBLIC_API_BASE_URI}
```

Comment:
They may not need these environment variables as they use a different backend now.

Reply:
Can you please let me know if there is anyone that I can confirm these with?

Reply:
As far as I know, there would be no one needed in this repo at this point. But maybe it is better just to leave them there to see if they are needed in the future.

Reply:
No change is needed at this point I believe.

Reply:
OK, then I will leave them there. Thanks!